examples/cpp/pyperf/PyPerfBPFProgram.cc

```diff
@@ -550,6 +553,7 @@ int read_python_stack(struct pt_regs* ctx) {
         &state->frame_ptr, sizeof(state->frame_ptr),
         cur_frame + state->offsets.PyFrameObject.f_back);
     if (!state->frame_ptr) {
+      event->kernel_stack_id = stack_traces.get_stackid(ctx, 0);
```

Comment:
Should move to around `submit`, because I wanna get kernel stacks even if unwinding had an error at whatever point.

Reply:
Done